### PR TITLE
New sites

### PR DESCRIPTION
**Pull request by 3xp0rt**

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -3767,6 +3805,16 @@
...
        },
+       {
+        "name" : "myWishBoard",
+        "uri_check" : "https://mywishboard.com/@{account}",
+        "e_code" : 200,
+        "e_string" : "class=\"MwbUserHeader\"",
+        "m_string" : "class=\"MwbError\"",
+        "m_code" : 404,
+        "known" : ["ke7_2024", "jpsol"],
+        "cat" : "hobby"
```
Comment (WebBreacher) [inline wmn-data.json:3816]: I believe this site should be `shopping`

File: wmn-data.json [hunk 2]
```diff
@@ -3767,6 +3805,16 @@
...
         "cat" : "xx NSFW xx"
        },
+       {
+        "name" : "myWishBoard",
+        "uri_check" : "https://mywishboard.com/@{account}",
+        "e_code" : 200,
+        "e_string" : "class=\"MwbUserHeader\"",
+        "m_string" : "class=\"MwbError\"",
+        "m_code" : 404,
+        "known" : ["ke7_2024", "jpsol"],
```
Comment (WebBreacher) [inline wmn-data.json:3815]: Please check to see if `jpsol` works for you. I get an error. The other account works.

**Review by WebBreacher — CHANGES_REQUESTED** [review]
Please check comments and alter as needed. Thanks for all the work!!!

File: wmn-data.json [hunk 3]
```diff
@@ -775,13 +775,14 @@
...
-        "name" : "Bugcrowd",
-        "uri_check" : "https://bugcrowd.com/{account}",
+        "name" : "BugCrowd",
+        "uri_check" : "https://bugcrowd.com/{account}/profile_widgets",
+        "uri_pretty" : "https://bugcrowd.com/{account}",
         "e_code" : 200,
-        "e_string" : "s researcher profile on Bugcrowd",
-        "m_string" : ">Bugcrowd | Error",
+        "e_string" : "\"widgets\":",
+        "m_string" : "class=\\'cc-error-page__msg\\'",
```
[the remark this answers is not in the document]
Comment (3xp0rt) [reply wmn-data.json:783]: Yes, I missed it during debugging.

**Review by WebBreacher — APPROVED** [review]

**Pull request merged**